tests/framework/helpers/HtmlTest.php

```
@@ -1521,6 +1521,25 @@ public function testActiveTextInput($value, array $options, $expectedHtml): void
+            'hidden' => ['activeHiddenInput', '<input type="hidden" id="htmltestmodel-name" name="HtmlTestModel[name]" value="">'],
+        ];
+    }
+
+    /**
+     * @dataProvider dataProviderActiveInputArrayValue
+     */
+    public function testActiveInputRendersEmptyForArrayValue($method, $expectedHtml): void
+    {
+        $model = new HtmlTestModel();
+        $model->setAttributes(['name' => ['tampered@example.com']], false);
+        $this->assertSame($expectedHtml, Html::$method($model, 'name'));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\helpers\Html' in method 'testActiveInputRendersEmptyForArrayValue'. (undefined)

(StaticAccess)